City-of-Helsinki/drupal-hdbt
# UHF-9497

# [UHF-9497](https://helsinkisolutionoffice.atlassian.net/browse/UHF-9497)


## What was done


* Removed link button classes from banner and hearings paragraphs.
* Refactored link-button twig to support new hds-button with data-variables.
* Moved 'open in new tab' helper text to helfi-link twig and added a check for data-variables in url object attributes.
* Removed obsolete class from buttons and converted banner to use data-variables.
* Fixed icon inclusions in banner paragraph.

## How to install and test

Check instructions from https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/690

## Designers review



## Other PRs


* https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/690
* https://github.com/City-of-Helsinki/drupal-helfi-strategia/pull/588
* https://github.com/City-of-Helsinki/drupal-helfi-asuminen/pull/362
* https://github.com/City-of-Helsinki/drupal-helfi-kuva/pull/299
* https://github.com/City-of-Helsinki/drupal-helfi-kasvatus-koulutus/pull/577
* https://github.com/City-of-Helsinki/drupal-helfi-rekry/pull/413
* https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/769
* https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/803
* https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/533
* https://github.com/City-of-Helsinki/drupal-helfi-tyo-yrittaminen/pull/350

[UHF-9497]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-9497?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
